src/components/pages/helper/googlemap.js

```diff
@@ -1,27 +1,108 @@
-import React, { Component } from 'react';
-import { Map, GoogleApiWrapper } from 'google-maps-react';
+import React, {Component } from 'react';
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
+import Pagination from "react-bootstrap/Pagination";
 
 const mapStyles = {
-  width: '100%',
-  height: '100%'
+  width: '100vw',
+  height: '60vh'
 };
 
 export class MapContainer extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            items: [
+                {
+                    "items": [
+                        {
+                            "name": "Freddy Fuego Hausmannsgate",
+                            "category": "food",
+                            "id": 0,
+                            "lat": 59.917792,
+                            "lng": 10.754015
+                        },
+                        {
+                            "name": "Venedig Pizzeria Restaurang & Café",
+                            "category": "food",
+                            "id": 1,
+                            "lat": 56.045759,
+                            "lng": 12.697704
+                        },
+                        {
+                            "name": "Morten Volunteer",
+                            "category": "volunteer",
+                            "id": 2,
+                            "lat": 56.045759,
+                            "lng": 12.697704
+                        }
+                    ]
+                }
+            ]
+        };
+    }
+
   render() {
+      const { items } = this.state;
+      const test = items[0].items;
+
+      let pins = [];
+
+      for(let i =0; i < 10; i++) {
+          if(test[i]){
+              pins.push(
+                  <Pagination.Item key={i}>
+                  </Pagination.Item>
+              )
+          }
+      }
```

Comment:
Why not simplify this to something like:

```
const pins = test.map( ( item, index ) => (
  <Pagination.Item key={ index }></Pagination.Item>
))
```

Also, those filters in the UI are intended to be fixed filters by category, not by specific location instance. Thoughts ?

Reply:
Ah right! I should have thought of that. Yes the filters are there to filter by category in the current map. If there’s a category missing it could also be left out from the list.